priv/static/mods/pao_rinshan.jq

```diff
@@ -0,0 +1,26 @@
+.after_call.actions += [
+  ["when", 
+    [
+      {"name": "status", "opts": ["rinshan_pao_possible"]},
+      {"name": "match", "opts": [["last_call"], [[[["quad"], 1]]]]}
+    ],
+    [["as", "callee", [["set_status", "pao_possible"]]]]],
+  ["unset_status_all", "rinshan_pao_possible"]
+]
+|
+# If anyone has pao_possible and anyone has just kan'd into a win, the player with pao_possible gets pao'd.
+.before_win.actions += [
+  ["when_anyone", [{"name": "status", "opts": ["pao_possible"]}, {"name": "anyone_status", "opts": ["kan"]}], ["set_status", "pao"]]
```

Comment:
if you are checking kan status above when setting pao_possible, this condition should always be true, so (i think) it's enough to set "pao" status directly rather than set pao_possible, check for pao_possible, then set pao

Reply:
in that case i guess pao should be unset after each turn change